src/plugins/plan/inspector/components/PlanActivitiesView.vue

```diff
@@ -20,21 +20,40 @@
  at runtime from the About dialog for additional information.
 -->
 <template>
-  <div class="c-inspector__properties c-inspect-properties">
-    <plan-activity-view
+  <div>
+    <plan-activity-time-view
       v-for="activity in activities"
-      :key="activity.id"
+      :key="activity.key"
+      class="c-inspector__properties c-inspect-properties"
```

Comment:
we should move these classes to the first element of each of these components since they won't be used in any other context, presumably.